source/modules/collection-api/address/overview.md.erb

```diff
@@ -21,6 +21,6 @@ street_number     | Yes       | String | The street number of the address
 zip_code          | Yes       | String | The zip code (*e.g. CEP*) of the address
 city              | Yes       | String | The city in which the address is located
 state             | Yes       | String | The state in which the address is located
```

Comment:
I suppose this would also nee to be a two-character ISO 3166-2 state code, right?

Reply:
Good catch!